framework/src/modules/chain/init_steps/create_http_server.js

```diff
@@ -0,0 +1,110 @@
+const express = require('express');
+const http = require('http');
+const socket = require('socket.io');
+const fs = require('fs');
+
+// There is an issue with promisify server.listen so used constructor
+const startServer = async (server, port, host) =>
+	new Promise((resolve, reject) => {
+		server.listen({ host, port }, err => (err ? reject(err) : resolve()));
+	});
+
+module.exports = async ({ config, components: { logger } }) => {
+	const app = express();
+
+	if (config.coverage) {
+		// eslint-disable-next-line import/no-extraneous-dependencies
+		const im = require('istanbul-middleware');
```

Comment:
are we still using this? (probably out of scope of this PR)

Reply:
That can be quested in seperate PR, where @yatki is moving to `nyc`, may be after that it can be removed.

Reply:
@nazarhussain I agree that we should get rid of it. However, I didn't fix it with #2858. we need another issue because we need to modify the way we instantiate the application in functional tests.

Reply:
@yatki Please create the related issue

Reply:
@shuse2 @nazarhussain created https://github.com/LiskHQ/lisk/issues/2919